CodeListLibrary_project/clinicalcode/api/views/Concept.py

```diff
@@ -986,6 +1000,10 @@ def getConcepts(request, is_authenticated_user=True, pk=None, set_class=Concept)
         tags, filter_cond = apply_filter_condition(query='tags', selected=tag_ids, conditions=filter_cond)
         collections, filter_cond = apply_filter_condition(query='tags', selected=collection_ids, conditions=filter_cond)
         
+    coding, filter_cond = apply_filter_condition(query='coding_system', selected=coding_ids, conditions=filter_cond)
+    sources, filter_cond = apply_filter_condition(query='coding_system_id', selected=data_sources, conditions=filter_cond)
```

Comment:
we don't have data sources in concept
and it is based on query='coding_system_id'